src/server/symbols.map

```diff
@@ -900,16 +908,678 @@ MIR_SERVER_2.17 {
     vtable?for?mir::shell::WindowManagerTools;
     vtable?for?mir::time::Alarm;
     vtable?for?mir::time::AlarmFactory;
-    mir::shell::ShellWrapper::surface_ready*;
-    mir::shell::ShellWrapper::set_popup_grab_tree*;
-    mir::DefaultServerConfiguration::the_main_clipboard*;
-    mir::DefaultServerConfiguration::the_primary_selection_clipboard*;
-    "mir::DefaultServerConfiguration::the_drag_icon_controller()";
-    "mir::DefaultServerConfiguration::the_pointer_input_dispatcher()";
-    mir::shell::SurfaceStackWrapper::swap_z_order*;
-    mir::shell::SurfaceStackWrapper::send_to_back*;
-    mir::shell::ShellWrapper::swap_z_order*;
-    mir::shell::ShellWrapper::send_to_back*;
- };
- local: *;
+  };
+local: *;
 };
+
+MIR_SERVER_2.18 {
```

Comment:
These look like they're meant to be the internal symbols?

Reply:
These are also "INTERNAL" symbols - and should also be in the `MIR_SERVER_INTERNAL_2.17` stanza